masterlist.yaml

```
@@ -3759,6 +3759,61 @@ plugins:
           - '[Extended Dialogue Interface](https://www.nexusmods.com/fallout4/mods/27216)'
           - '[SS2 - XDI Compatibility Patch](https://www.nexusmods.com/fallout4/mods/48254)'
         condition: 'active("XDI.esm") and not active("SS2_XDI Patch.esp") and not active("SS2_XDI Patch-Basic.esp") and not active("SS2_XDI Patch-Basic_esl.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Horizon V1.8](https://www.nexusmods.com/fallout4/mods/17374)'
+          - '[Sim Settlements 2 - Horizon Patch](https://www.nexusmods.com/fallout4/mods/54186)'
```

You can leave the version number out of the mod title since it's subject to change, & the links should have trailing slashes:

```suggestion
          - '[Horizon](https://www.nexusmods.com/fallout4/mods/17374/)'
          - '[Sim Settlements 2 - Horizon Patch](https://www.nexusmods.com/fallout4/mods/54186/)'
```